src/components/atoms/VerifiedPublisher.module.css

```diff
@@ -0,0 +1,31 @@
+.loader,
+.verifiedBadge {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: calc(var(--spacer) / 3);
+  height: calc(var(--spacer) * 1.5);
+}
+
+.verifiedBadge {
+  border-radius: var(--border-radius);
+
+  background: var(--color-primary);
+  color: var(--brand-white);
+
+  font-weight: var(--font-weight-bold);
+}
+
+.verifiedBadge svg {
+  display: inline-block;
+  height: 100%;
+}
+
+.verifiedBadge svg path {
+  fill: var(--brand-white) !important;
```

Comment:
nitpick(blocking): we should avoid this pattern as suggested by MDN: https://developer.mozilla.org/en-US/docs/MDN/Guidelines/Code_guidelines/CSS#dont_use_!important
is there a different way to achieve this?